### PR TITLE
Do not always pull ROKS toolkit images

### PR DESCRIPTION
**Pull request by SmithaSubbarao**
Related to this issue: https://github.ibm.com/alchemy-containers/armada-update/issues/3299

**DNM until all updates have been made.**

### CODE REVIEW

File: .secrets.baseline [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "files": "vendor/*|go.sum|^.secrets.baseline$",
     "lines": null
   },
-  "generated_at": "2022-05-16T13:33:33Z",
+  "generated_at": "2022-05-17T19:53:06Z",
```
Comment (rtheis) [inline .secrets.baseline:6]: Please do not update the secrets baseline since it will block cherry-picks.

**Review by rtheis — CHANGES_REQUESTED** [review]
This PR doesn't have any changes to the imagePullPolicy configuration for actual deployments.  The only change is to an example yaml.

File: assets/cluster-version-operator/cluster-version-operator-deployment.yaml [hunk 2]
```diff
@@ -54,7 +54,7 @@ spec:
       containers:
         - name: cluster-version-operator
           image: {{ .ReleaseImage }}
-          imagePullPolicy: Always
+          imagePullPolicy: IfNotPresent
```
Comment (rtheis) [inline assets/cluster-version-operator/cluster-version-operator-deployment.yaml:57]: Please search for `imagePullPolicy: Always` and you will find more parts to change.

**Review by rtheis — CHANGES_REQUESTED** [review]

File: assets/cluster-version-operator/cluster-version-operator-deployment.yaml (hunk 2 above)
Comment (SmithaSubbarao) [reply assets/cluster-version-operator/cluster-version-operator-deployment.yaml:57]: I've searched the entire repo for more images with ```imagePullPolicy: Always```, but I can't seem to find any. My fork is updated, but I'll check again. I'm sure there will be more to change.

Comment (rtheis) [reply assets/cluster-version-operator/cluster-version-operator-deployment.yaml:57]: See https://github.com/openshift/ibm-roks-toolkit/blob/master/assets/roks-metrics/roks-metrics-deployment.yaml#L38 and https://github.com/openshift/ibm-roks-toolkit/blob/master/assets/roks-metrics/roks-metrics-push-gateway-deployment.yaml#L39

**Review by rtheis — CHANGES_REQUESTED** [review]

File: assets/cluster-version-operator/cluster-version-operator-deployment.yaml (hunk 2 above)
Comment (SmithaSubbarao) [reply assets/cluster-version-operator/cluster-version-operator-deployment.yaml:57]: Thank you Richard, I realized that I changed those files but didn't save them, which is why they didn't show up when I searched for them.

**Review by rtheis — APPROVED** [review]
/lgtm

**Review by rtheis — APPROVED** [review]
/approved

**Comment by rtheis** [comment]
@SmithaSubbarao have you completed your testing?

**Comment by rtheis** [comment]
@SmithaSubbarao I doubt this PR would cause such failures.  Remember that you cannot use the same ROKS toolkit version for all ROKS versions. They each need a separate build.

**Review by rtheis — CHANGES_REQUESTED** [review]
@SmithaSubbarao please remove `ROKS 4.10` from the PR description.

**Comment by SmithaSubbarao** [comment]
Fixing the commit for this PR

**Comment by SmithaSubbarao** [comment]
Closing this PR in favor of a new PR.

**Pull request closed without merging**